Dshop/apps/products_catalogue/tests.py

```diff
@@ -15,10 +15,3 @@ def test_create_category_with_parent():
     parent_category = Category.objects.create(name='Parent Category', is_active=True)
     child_category = Category.objects.create(name='Child Category', is_active=True, parent=parent_category)
     assert child_category.parent == parent_category
-
-
-@pytest.mark.django_db
-def test_different_slugs_for_same_category_name():
-    category1 = Category.objects.create(name='Main Category', is_active=True)
-    category2 = Category.objects.create(name='Main Category', is_active=True)
-    assert category1.slug != category2.slug
```

Comment:
This test is not valid anymore and there is no case we could test, at least now.